jigsaw

```diff
@@ -47,13 +51,23 @@ $container->bind(BladeHandler::class, function ($c) {
     return new BladeHandler($c[Factory::class]);
 });
 
+$container->bind(MarkdownParser::class, \Mni\FrontYAML\Bridge\Parsedown\ParsedownParser::class);
+$container->bind(YAMLParser::class, \Mni\FrontYAML\Bridge\Symfony\SymfonyYAMLParser::class);
+$container->bind(Parser::class, function ($c) {
+    return new Parser($c[YAMLParser::class], $c[MarkdownParser::class], '---', '---');
+});
```

Comment:
The two separators `'---'` aren't necessary, FrontYAML defines those by default in its constructor.